utils/libnetwork/Makefile

```diff
@@ -1,7 +1,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libnetwork
-PKG_VERSION:=0.8.0-dev.2
+PKG_SOURCE_VERSION:=e7933d41
+PKG_VERSION:=0.8.0-dev.2-${PKG_SOURCE_VERSION}
 PKG_RELEASE:=1
 PKG_LICENSE:=Apache-2.0
```

Comment:
Move below PKG_MAINTAINER. Same with _FILES.

Reply:
Could you please direct me to the style guide that prefers this?
The only thing I could find resembling one was [the packages page in the docs](https://openwrt.org/docs/guide-developer/packages#buildpackage_variables) which might suggest via the ordering that the current layout is preferred, however it looks like it might be outdated.

Reply:
https://github.com/openwrt/packages/issues/9682

Reply:
Thanks, I look forward to the possibility of a linter to resolve these conversations in the future.